### DISCUSSION TITLE
Can someone help me with validation?

### DISCUSSION THREAD

**Discussion by YolkKanapon** [react-hook-form/react-hook-form#3962] [Q&A]
The problem is my validation always alert, even I already choose a selection
`<Controller
                        control={control}
                        name="branch"
                        render={() => (
                        <Dropdown
                            label={en.pages.employee.branch}
                            onChange={handleInputChange}
                            value={values.branch}
                            name="branch"
                            options={service.dataFromService}
                        />)}
                        errors={errors.branch}
                        rules={{ required: true }}
                    />
                    {errors.branch && <FormHelperText>{en.validate.selectRequire}</FormHelperText>}`

Here's my select ref [https://codesandbox.io/s/react-hook-form-select-onchange-forked-eqb20?file=/src/App.js:1754-1766](url)

**Comment by bluebill1049** [comment]
perhaps you want to use `validate` function to see, maybe you are trying to validate an object, which is always going to pass required.
